services/brig/test/integration/API/Provider.hs

```diff
@@ -558,8 +575,8 @@ testDeleteService config db brig = do
     getServiceProfile brig uid pid sid !!!
         const 404 === statusCode
 
-testAddRemoveBot :: Maybe Config -> FilePath -> DB.ClientState -> Brig -> Galley -> Cannon -> Http ()
```

Comment:
The certificate path can be taken from the config so there's no need to pass it around